src/jobs/veracode_policy_scan.yml

```diff
@@ -12,25 +17,17 @@ parameters:
     description: |
                     Comma-separated list of team names associated with the specified application. Validates against the names of existing teams for this account.
     type: string
-  docker_image_app_dir:
-    description: Directory inside the docker image where the application artifacts are saved
-    type: string
-    default: "/app"
 steps:
   - veracode_prepare_artifacts:
```

Comment:
Does this step work when run in docker  image`veracode/api-wrapper-java:latest`  ?